### PR TITLE
[Feat]:List component added with 2 variations

### PR DESCRIPTION
**Pull request by t007rushi**
LIST Component added
-> Normal list with single-line text

-> Stacked list with horizontal & vertical stacked data in row
**A reviewer left a comment:**

screenshot

<details><summary>Browser metadata</summary>

```
Path:      /documentation/document#list
Browser:   Chrome 98.0.4758.102 on Windows 10
Viewport:  2133 x 1076 @0.8999999761581421x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-12--pushpa-ui.netlify.app%2Fdocumentation%2Fdocument%23list&os=Windows&os_version=10&browser=Chrome&browser_version=98.0&resolution=2133x1076&speed=1&start=true&ref=netlify-source)

</details>

[Open Deploy Preview](https://deploy-preview-12--pushpa-ui.netlify.app/documentation/document#list) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6212bfd9295abc000821bba9&commentID=6212c10d335c6ada58518ed9&resolution=resolved)

### CODE REVIEW

File: components/list.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  text-align: center;
+  padding: 10px;
+  font-size: 1.2rem;
+  color: var(--cta-color);
+}
+
+.norm-list-item > div {
+  color: var(--primary-color);
+  background-color: var(--secondary-color);
+  border-radius: 50%;
```
Comment (dev-enforced) [inline components/list.css:20]: If using this value frequently then you can use this as a variable named as --bd-rd-circle:50% and use it accordingly.

**Review by dev-enforced — APPROVED** [review]

File: components/list.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.list-container{
+width: 18rem;
+/* height: 8rem; */
+background-color:var(--primary-color);
+margin:1rem;
+text-align:center;
+/* padding:10px; */
```
Comment (Priya-Patel31) [inline components/list.css]: remove these commented text

File: documentation/document.html [hunk 3]
```diff
@@ -556,6 +556,70 @@ <h1 class="comp-title">Rate Product</h1>
           style="width: 100%; height: 281px"
           frameborder="0"
         ></iframe>
+                <!-- LISTS -->
+                 <!-- NORMAL LIST -->
+                 <div class="flex-col center-it">
+                  <h1>LISTS</h1>
```
Comment (Priya-Patel31) [inline documentation/document.html]: give some margin across y-axis as in UI heading seems closer with above code.

File: documentation/document.html [hunk 4]
```diff
@@ -556,6 +556,70 @@ <h1 class="comp-title">Rate Product</h1>
...
+                      <div class="flex-col gap-btwn">
+                        <span class="material-icons"> feed </span>1 hour ago
+                      </div>
+                      <div class="flex-col gap-btwn">
+                        Top story
+                        <p class="stack-list-text">
+                          Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        </p>
+                      </div>
+                      <img src="../ignore/modi.jfif" alt="" class="stk-list-img" />
```
Comment (Priya-Patel31) [inline documentation/document.html]: give some alternate text in alt tag

File: documentation/document.html [hunk 5]
```diff
@@ -556,6 +556,70 @@ <h1 class="comp-title">Rate Product</h1>
...
+                        <span class="material-icons"> feed </span>1 hour ago
+                      </div>
+                      <div class="flex-col gap-btwn">
+                        Top story
+                        <p class="stack-list-text">
+                          Est perspiciatis beatae facilis voluptatem iste?
+                          Necessitatibus corrupti minus
+                        </p>
+                      </div>
+                      <img src="../ignore/river.jfif" alt="" class="stk-list-img" />
```
Comment (Priya-Patel31) [inline documentation/document.html]: same as above !!

File: documentation/document.html [hunk 6]
```diff
@@ -556,6 +556,70 @@ <h1 class="comp-title">Rate Product</h1>
...
+                        <span class="material-icons"> feed </span>1 hour ago
+                      </div>
+                      <div class="flex-col gap-btwn">
+                        Top story
+                        <p class="stack-list-text">
+                          nulla blanditiis, quia, numquam nihil quo facilis, amet id
+                          recusandae? Nihil, cum aspernatur?
+                        </p>
+                      </div>
+                      <img src="../ignore/cricket.jfif" alt="" class="stk-list-img" />
```
Comment (Priya-Patel31) [inline documentation/document.html]: Just a suggestion that you can use class name  as stack-list-img rather than stk-list-img

**Review by SuyashTiwariCoder — APPROVED** [review]
Looks good to me , You can merge .

**Review by Priya-Patel31 — APPROVED** [review]
Just need to make some minor changes in the code rest everything seems good .

File: components/list.css (hunk 1 above)
Comment (t007rushi) [reply components/list.css:20]: nice idea to add utility thank you for suggestion

**Comment by t007rushi** [comment]
> Just need to make some minor changes in the code rest everything seems good .

thank you for pointing out these minor changes it helps

**Pull request merged**